compositor_render/src/transformations/builtin/params/tiled_layout.rs

```diff
@@ -232,16 +178,18 @@ impl TiledLayoutParams {
 
         best_rows_cols
     }
+
+    fn transformation_matrix(
+        tile_layout: &BoxLayout,
+        input_resolution: Resolution,
+        output_resolution: Resolution,
+        tile_size: Resolution,
+    ) -> Mat4 {
+        let fit = FitParams::new(input_resolution, tile_size).scale_matrix;
```

Comment:
It's a little bit hacky, since fit only changes the aspect ratio, while the box layout transforms and scales it to the correct resolution. I'm planning to create a little more advanced wrapper for those transformations, addressing this issue